pkg/skaffold/render/transform/transform.go

```diff
@@ -86,7 +108,7 @@ func validateTransformers(config []latestV2.Transformer) ([]kptfile.Function, er
 		}
 		if c.ConfigMapData != nil {
 			for _, stringifiedData := range c.ConfigMapData {
-				items := strings.Split(stringifiedData, "=")
+				items := strings.Split(stringifiedData, ":")
 				if len(items) != 2 {
 					return nil, sErrors.NewErrorWithStatusCode(
 						proto.ActionableErr{
```

Comment:
nit: have all the errors in an `errors.go` file. Like [this](https://github.com/gsquared94/skaffold/blob/823896bdd7464d79886664b32f56948957ea5f65/pkg/skaffold/schema/errors/errors.go) one.

Reply:
Gotcha. Done.

Reply:
I meant creating a new file `errors.go` in the `render` package. The one I linked was only as an example.

Reply:
I see. Done.